### PR TITLE
Review one of my past projects ⚒️

### PR DESCRIPTION
**Pull request by ritikarawat220**
Microverse-Project-for-trials
-Add index.html and style.css file
-Review one of my past projects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <p>The man who saved a billion lives</p>
+        <div class="image-container">
+            <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="tribute image">
+            <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+        </div>
+
+    <!-- 
+    <div>
+      <h3>Hello Everyone, we are software developers!</h3>
+    </div>  -->
```
Comment (ritikarawat220) [inline index.html:22]: Please do not leave unused code as comment in your codebase. Unused code as comments makes your code not clean.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        </ul>
+        <ul>
+            <li><span>1962 </span>- Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</li>
+            <li><span>1970 </span>- receives the Nobel Peace Prize</li>
+            <li><span>1983 </span>- helps seven African countries dramatically increase their maize and sorghum yields</li>
+            <li><span>1984 </span>- becomes a distinguished professor at Texas A&M University</li>
+            <li><span>2005 </span>- states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds."</li>
+            <li><span>2009 </span>- dies at the age of 95.</li>
+        </ul>
```
Comment (ritikarawat220) [inline index.html:48]: Please create this dynamically and put this in one ul group, so as to not confuse others and also for more code readability and maintainability.

**Review by ritikarawat220 — COMMENTED** [review]
# CHANGES REQUIRED ♻️ 

image

Please check my comments below and make all required changes.

Submit for a re-review when you're done. 😄 

Thanks 🤗

**Review by Owusu-Desmond — APPROVED** [review]
Hi @ritikarawat220  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Good to use inline comments :heavy_check_mark: 
📌 Great working on the setup of the project ✅
📌 Good communication ✅
📌 Good commit history ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**